src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -0,0 +1,148 @@
+@observer
+export default class GroupComparisonMutationsTabPlot extends React.Component<
+    IGroupComparisonMutationsTabPlotProps,
+    {}
+> {
+    @observable public showPercent = true;
+    constructor(props: IGroupComparisonMutationsTabPlotProps) {
+        super(props);
+        makeObservable(this);
+    }
+
+    private mutationMapperToolStore = remoteData<MutationMapperToolStore>({
+        await: () => [this.props.store.mutationsByGroup],
+        invoke: () => {
+            const store = new MutationMapperToolStore(this.props.mutations, {
+                ...this.props.filters,
+                countUniqueMutations: this.countUniqueMutations,
+            });
+            return Promise.resolve(store);
+        },
+    });
+
+    @autobind
+    protected countUniqueMutations(mutations: Mutation[]) {
+        return this.showPercent
+            ? (countUniqueMutations(mutations) /
+                  this.props.store.profiledSamplesCount.result!) *
+                  100
+            : countUniqueMutations(mutations);
+    }
+
+    @action.bound
+    private onScaleToggle(selectedScale: AxisScale) {
+        this.showPercent = selectedScale === AxisScale.PERCENT;
+    }
+
+    @autobind
+    protected plotLollipopTooltipCountInfo(
+        count: number,
+        mutations?: Mutation[]
+    ): JSX.Element {
+        const decimalZeros = numberOfLeadingDecimalZeros(count);
+        const fractionDigits = decimalZeros < 0 ? 1 : decimalZeros + 2;
+        return mutations && mutations.length > 0 && this.showPercent ? (
+            <strong>
+                {formatPercentValue(count, fractionDigits)}% mutation rate
+            </strong>
+        ) : (
+            <strong>
+                {count} mutation{`${count !== 1 ? 's' : ''}`}
+            </strong>
+        );
+    }
+
+    protected plotYAxisLabelFormatter(symbol?: string, groupName?: string) {
+        return `${symbol} ${groupName} Mutations`;
+    }
+
+    readonly plotUI = MakeMobxView({
+        await: () => [
+            this.props.store.mutations,
+            this.props.store.mutationsByGroup,
+            this.props.store.profiledSamplesCount,
+            this.mutationMapperToolStore,
+        ],
+        render: () => {
+            const mutationMapperStore = this.mutationMapperToolStore.result!.getMutationMapperStore(
+                this.props.store.selectedMutationMapperGene!.hugoGeneSymbol
+            );
+            return (
+                <div>
+                    <Loader
+                        isLoading={
+                            this.mutationMapperToolStore.result!
+                                .mutationMapperStores.isPending &&
+                            this.props.store.mutations.isPending &&
+                            this.props.store.mutationsByGroup.isPending &&
+                            this.props.store.profiledSamplesCount.isPending
+                        }
+                    />
+                    {mutationMapperStore &&
```

Comment:
you can probably assume you have mutationmapperStore since you are awaiting them above